lib/routes/routesUtils.js

```diff
@@ -541,4 +541,22 @@ const routesUtils = {
     },
 };
 
+// Queries supported by AWS that we do not currently support.
```

Comment:
I don't want to mostly because it would mean the unsupportedQueries object would then be imported wherever else routeUtils is used, e.g. all the routes. It's not a big object but it is unnecessary given that the only place we use it so far is in routes.js. What do you think?